index.js

```diff
@@ -0,0 +1,38 @@
+const {
```

Comment:
index.js is a proper name for this file ?

Reply:
Ah, Sorry, index.js is the default entry file for a package unless we define `main` field in the package.json.
I think it would be better if we could follow the same convention as we did in `@scalar-labs/scalardl-web-client-sdk`.
I will rename it to be `scalardl-node-client-sdk.js` and set this name in the package.json.